workLoggingSystem.html

```diff
@@ -339,10 +339,13 @@
                     </div>
                     <div class="row formComponent">
                         <div class="col"></div>
-                        <div class="col-md-2" id="createNewTaskSubmit">
-                            <button id="submitNewTask">Submit</button>
+                        <div class="col-md-3" id="createNewTaskSubmit">
+                            <div class="row">
+                            <div class="col-md-10"><p id="errorMessage">*Please enter the valid details</p></div>
+                            <div class="col-md-10"><button id="submitNewTask">Submit</button></div>
```

Comment:
wrong and incorrect implementation of bootstrap, please learn bootstrap grid system